templates/tutorialv2/includes/content/content.part.html

```diff
@@ -15,7 +15,7 @@
         <p>
             {% trans "Il n’y a pas d’introduction." %}
             {% if display_config.draft_actions.enable_edit %}
-                {% trans "Vous pouvez " %}<a href="{% url "content:edit" content.pk content.slug %}">{% trans "en ajouter une" %}</a>.
+                {% trans "Vous pouvez " %}<a href="{% url "content:edit-introduction" content.pk %}">{% trans "en ajouter une" %}</a>.
             {% endif %}
         </p>
     </div>
```

Comment:
```suggestion
    </div>
    
    <hr>
```

Il y a une séparation entre le bandeau pour ajouter une section ou partie et le bandeau pour ajouter une conclusion. Je trouve ça mieux d'en mettre une aussi pour séparer le bandeau pour ajouter une introduction. Uniquement lorsque le bandeau est affiché, car sinon ça fera bizarre entre l'introduction et le sommaire.

Reply:
C'est fait.